items/resources.py

```diff
@@ -1,14 +1,47 @@
-from tastypie.resources import ModelResource
+from tastypie.resources import ModelResource, Resource
 from tastypie import fields
 from tastypie.authorization import Authorization
+from tastypie.authentication import SessionAuthentication
+
+from django.contrib.sessions.models import Session
+
+from  django.conf.urls import url
+from django.contrib import auth
+from django.http import HttpResponse
 
 from items.models import Item
 from django.contrib.auth.models import User
 
+import json
+
+class UserAuthorization(Authorization):
+
+  def get_user(self, bundle):
+    return bundle.request.user
+
+  def read_list(self, object_list, bundle):
+    return object_list.filter(user=self.get_user(bundle))
+
+  def read_detail(self, object_list, bundle):
+    return bundle.obj.user == bundle.request.user
+
+  def update_detail(self, object_list, bundle):
+    return bundle.obj.user == bundle.request.user
+
+  def create_detail(self, object_list, bundle):
+    bundle.obj.user = self.get_user(bundle)
+    return True
+
+  def delete_detail(self, object_list, bundle):
+    return bundle.obj.user == bundle.request.user
+
+
+
 class UserResource(ModelResource):
   class Meta:
     queryset = User.objects.all()
     resource_name = 'user'
+    excludes = ['email', 'password', 'is_active', 'is_staff', 'is_superuser']
```

Comment:
Also it is a good thing to keep authorization here - to allow full access for superuser and limit access for logged users to their entities only.